velox/exec/SetAccumulator.h

```diff
@@ -86,6 +86,42 @@ struct SetAccumulator {
     }
   }
 
+  void addNullIndex(const char* buffer) {
+    vector_size_t serializedNullIndex;
+    memcpy(&serializedNullIndex, buffer, kVectorSizeT);
+    if (!nullIndex.has_value() && serializedNullIndex != kNoNullIndex) {
```

Comment:
Is it possible for nullIndex.has_value() to be true here? If not, let's remove this check. Otherwise, move it before memcpy.